CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Timelines
 
+## 0.3.3 (2024-05-27)
```

Comment:
This should probably bump to 0.3.0, as we should reset the `patch` digit (we probably should have done that at 0.2.x too, but I didn't notice)